src/static/js/form.js

```diff
@@ -14,9 +12,25 @@ function tableUpdate(user) {
         '<button type="submit" name="delete" class="btn btn-danger">Delete</button></form></th>'
 
     row += '</tr>';
-    alert(row);
```

Comment:
Could: Replace  'alert' on console.log(row) or even console.dir(row). It is more convenient